packages/react-components/src/components/SystemMessage/SystemMessage.tsx

```
@@ -0,0 +1,233 @@
+          />
+        )}
+      </div>
+    );
+  };
+
+  const getDetails = (details: string[]): React.ReactNode => {
+    return (
+      <div data-testid="system-message-details" className={detailsClassName}>
+        <div className={detailsFirstLineClassName}>
+          {details.slice(0, 1).map((detail) => (
+            <Text as="div" size="xs">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)